src/components/SearchOverlay/SearchOverlay.js

```diff
@@ -230,77 +174,6 @@ class SearchOverlay extends React.PureComponent {
     return searchMode;
   };
 
-  isActiveResult = result => {
-    const { activeResult } = this.props;
-
-    if (!activeResult) {
-      return true;
-    }
-
-    const inSamePage = activeResult.pageNum === result.pageNum;
-    const hasSameCoordinates =
-      Object.values(activeResult.quads[0]).toString() ===
-      Object.values(result.quads[0]).toString();
-
-    return inSamePage && hasSameCoordinates;
-  };
-
-  executeSingleSearch = (isSearchUp = false) => {
```

Comment:
With new 7.0 UI we don't have way of executing single searches, thus removing the implementation